pages/forgot_password.php

```diff
@@ -0,0 +1,28 @@
+<?php
+setTitle("Wachtwoord vergeten");
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
```

Comment:
Je haalt hier eerst de email uit de database, en later pas kijk je of het email wel is ingevult.